SECURITY.MD

```diff
@@ -0,0 +1,35 @@
+<!-- BEGIN FLEDGE SECURITY.MD V0.0.1 BLOCK -->
+
+## Security
+
+FLEDGE takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [FLEDGE](https://github.com/fledge-iot)
+
+
+If you believe you have found a security vulnerability in any Fledge-owned repository please report it to us as described below.
```

Comment:
Fledge-owned ?

Reply:
Changed to Fledge repositoties instead of Fledge-owned.